cmake/StaticBuild.cmake

```diff
@@ -57,6 +57,8 @@ if(BUILD_STATIC)
       set(GNUTLS_LIBRARIES "${GNUTLS_LIBRARIES} -lcrypt32")
       # And sockets
       set(GNUTLS_LIBRARIES "${GNUTLS_LIBRARIES} -lws2_32")
+	  # And others
+      set(GNUTLS_LIBRARIES "${GNUTLS_LIBRARIES} -lp11-kit -lidn2 -lunistring")
```

Comment:
This unfortunately breaks some setups, including ours. Are these optional dependencies of GnuTLS?